autoPyTorch/pipeline/components/setup/network/utils/common.py

```diff
@@ -0,0 +1,53 @@
+from typing import Tuple
+
+import torch.nn as nn
+import math
+
+
+def initialize_weights(module: nn.Module) -> None:
```

Comment:
Don't we already have an initializer in https://github.com/LMZimmer/Auto-PyTorch_refactor/blob/development/autoPyTorch/pipeline/components/setup/network_initializer/base_network_initializer.py ?

Reply:
Yes, I will remove it from the network.